config.js

```diff
@@ -1,5 +1,10 @@
+const {
+  REQUEST_THROTTLE_MS,
+  REQUEST_TIMEOUT_MS,
+} = process.env;
+
 export default {
-  REQUEST_THROTTLE_MS: 5000,
-  REQUEST_TIMEOUT_MS: 5000,
+  REQUEST_THROTTLE_MS: Number(REQUEST_THROTTLE_MS) || 5000,
+  REQUEST_TIMEOUT_MS: Number(REQUEST_TIMEOUT_MS) || 5000,
```

Comment:
Yeah definitely. I was thinking we should increase this.

Though we should also expect increased memory overhead when it's under a lot of load I'd imagine?